VaticanBAV/et/BAVet117.xml

```diff
@@ -551,6 +553,8 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                       <msIdentifier><idno/></msIdentifier>
```

Comment:
the codicological description (physDesc) of the two units is not there yet

Reply:
Which details, aside from `<additions`, should be moved from the general `physDesc` to the `msPart`s' (if most of the details, like measurements and number of lines, are the same)?

Reply:
we regard each codicological unit as a manuscript of its own which may have potentially circulated on its own (probably not in this case but nonetheless)
so it should have extent/collation (not the full table but e.g. in this case the information on quire signatures)/measurements/layout/ etc. just as if you were describing two mss